packages/account-v2/src/components/responsive-wrapper/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment } from 'react';
+import { useBreakpoint } from '@deriv/quill-design';
+
+type TResponsiveWrapperProps = {
+    children: {
+        desktop: React.ReactNode;
+        mobile: React.ReactNode;
+    };
+};
+
+export const ResponsiveWrapper = ({ children }: TResponsiveWrapperProps) => {
```

Comment:
what is the purpose of this component? 
why just not to add in child component the checking if it is for desktop or responsive?
are there some specific scenarios requiring such wrapper?

Reply:
I thought a wrapper would be better to encapsulate this logic as I have the same logic in Stepper. Just a POC sort